docs/api.md

```diff
@@ -107,11 +117,33 @@ Results of detecting a virtual machine.
 
 `probability` - possible values = `[0.0 .. 1.0 | -1.0 in case of "error", "notEnoughData" statuses]`
 
-`type` - possible values = `"vmware"`, `"parallels"`, `"virtualBox"` or others
+`type` - optional field, possible values = `"vmware" | "parallels" | "virtualBox" | ... `
+
+***
+### `mode` is `requestId`:
+
+```js
+{
+    "requestId": "<requestId>"
+}
+```
+
+You can use method `BotDetector.poll` then to get full detection report by `requestId`.
+
+## `BotDetector.poll`
+
+```js
+botd.poll(): Promise<object>
+```
+
+The response will be the same as `BotDetector.get` has for `mode = "allData"`.
+The only one difference is that it's possible to receive `status = "inProgress"` if detection report is not ready yet.
+In this case try to call the method one more time after small period of time.
```

Comment:
This part is confusing, shouldn't poll handle retries + delays automatically?

Reply:
poll() with retries and delays was implemented https://github.com/fingerprintjs/botd/pull/13
Description fixed